# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/no-spp.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The dynamic flavour does not built for us due undefined __guard, but since
we do not care for the size optimized binaries we do just do not enable it
automatically for now.

  - Rene Rebe <rene@exactcode.de>

--- dietlibc-0.30/dietfeatures.h	2006-04-04 07:35:14.000000000 +0200
+++ dietlibc-0.30-fixed/dietfeatures.h	2006-06-30 15:38:05.000000000 +0200
@@ -108,7 +107,7 @@
  * WANT_SSP_URANDOM and then not using -fstack-protector enlarges a
  * binary by 152 bytes. */
 #if (__GNUC__>4) || ((__GNUC__==4) && (__GNUC_MINOR__>=1))
-#define WANT_SSP
+//#define WANT_SSP
 #endif
 /* Choose which canary seeder you want you can choose
  * both but it will check urandom first and use xor as
